webrob/neems/neem.py

```diff
@@ -11,25 +11,29 @@ class NEEM:
     def __init__(self,
                  neem_id):
         # collect neem by id
-        app.logger.info('getting neem from mongo with id: ')
-        app.logger.info(neem_id)
-        neem = mongoDBMetaCollection.find_one({"_id": neem_id})
-        app.logger.info(neem)
-        self.neem_id = neem['_id']
-        self.name = neem['name']
-        self.description = neem['description']
-        self.created_by = neem['created_by']
-        self.created_at = neem['created_at']
-        self.model_version = neem['model_version']
-        self.downloadUrl = neem['url']
-        self.knowrob_image = 'knowrob'
-        self.knowrob_tag = 'latest'
-        self.maintainer = neem['created_by']
-        self.authors = neem['created_by']
-        self.acknowledgements = ''
-        self.environments = ''
-        self.activities = ''
-        self.agents = ''
+        if neem_id is None:
+            return None
```

Comment:
does this have a meaning, returning None in a constructor?
never seen that before. Probably not a good idea.

Reply:
that would be more of a exception situation, or?

Reply:
Hi, yes you are right. It was temporary solution to see if we can make explore link make it work. So, I talked with sascha and he has made changes which asserts if neem_id is in desired format. so I am closing this pull request.